roxie/ccd/ccdmain.cpp

```diff
@@ -543,6 +543,8 @@ version: "1.0"
     - name: workunit
       port: 0
       numThreads: 0
+  logging:
+      detail: 100
```

Comment:
What does this correspond to, and how does it related to the existing traceLevel setting